src/runtimes/node/utils/entry_file.ts

```diff
@@ -26,7 +26,7 @@ const getEntryFileContents = (mainPath: string, moduleFormat: string, featureFla
 
   if (featureFlags.zisi_functions_api_v2) {
     return [
-      `import func from '${importPath}'`,
+      `import * as func from '${importPath}'`,
```

Comment:
`getLambdaHandler` expects an object with a `default property. Previously this seemed to work, because we were still transpilling the code